lib/subpackages.bzl

```diff
@@ -62,7 +62,10 @@ def _all(exclude = [], allow_empty = False, fully_qualified = True):
     return subs
 
 def _fully_qualified(relative_path):
-    return "//%s/%s" % (native.package_name(), relative_path)
+    package_name = (
+        "%s/" % native.package_name() if native.package_name() else ""
```

Comment:
This code is a bit hard to understand. On first read I though it was 
`"%s/" % (native.package_name() if native.package_name() else "")`

Consider something simpler like:
```
if native.package_name():
  return "//" + relative_path
else:
  return "//%s%s" % (package_name, relative_path)
```

Reply:
Ups, swapped then and else branch.

Reply:
@comius Thanks for your feedback. I changed the code. It should be better readable now.